script.module.codequick/addon.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="script.module.codequick" name="CodeQuick" provider-name="willforde" version="1.0.1+matrix.1">
+    <requires>
+        <import addon="xbmc.python" version="3.0.0"/>
+        <import addon="script.module.youtube.dl" version="19.912.1"/>
+        <import addon="script.module.htmlement" version="1.0.0"/>
+        <import addon="script.module.requests" version="2.22.0"/>
+    </requires>
+    <extension point="xbmc.python.module" library="lib"/>
+    <extension point="xbmc.addon.metadata">
+        <summary lang="en_GB">Framework for creating kodi add-on's.</summary>
+        <description lang="en_GB">Codequick is a framework for kodi add-on's. The goal of this framework is to simplify add-on development. This is achieved by reducing the amount of boilerplate code to a minimum, automating tasks like route dispatching and sort method selection. Ultimately allowing the developer to focus primarily on scraping content from websites and passing it to kodi.</description>
+        <license>GNU GENERAL PUBLIC LICENSE. Version 2, June 1991</license>
```

Comment:
GPL-2.0-only or GPL-2.0-or-later

Reply:
GPL-2.0-only is that ok.

Reply:
only you know, as long as it's an SPDX identifier for the license I'm fine :)

Reply:
I get you now. I was a bit confused. Changed it to GPL-2.0-only